pkg/apis/agones/v1/gameserver.go

```diff
@@ -356,6 +394,25 @@ func (gss *GameServerSpec) applySchedulingDefaults() {
 	}
 }
 
+func (gss *GameServerSpec) applyEvictionDefaults() {
+	if !runtime.FeatureEnabled(runtime.FeatureSafeToEvict) {
+		return
+	}
+	if gss.Eviction.Safe == "" {
+		gss.Eviction.Safe = EvictionSafeNever
+	}
+}
+
+func (gs *GameServer) applyEvictionStatus() {
+	if !runtime.FeatureEnabled(runtime.FeatureSafeToEvict) {
+		return
+	}
+	gs.Status.Eviction = gs.Spec.Eviction
```

Comment:
I missed this bit before - but if the value of status.eviction is never going to change... why do we have a status value for eviction?

Reply:
(responded in https://github.com/googleforgames/agones/pull/2857#discussion_r1062784460)